datarepo-clienttests/src/main/java/scripts/testscripts/CreateSnapshot.java

```diff
@@ -39,19 +46,41 @@ public CreateSnapshot() {
 
   private static List<BlobId> scratchFiles = new ArrayList<>();
 
+  private BlobIOTestUtility blobIOTestUtility;
+
+  private AzureResourceManager azureClient;
+
+  private BulkLoadArrayRequestModel arrayLoad;
+
+  private IngestRequestModel ingestRequest;
+
   public void setup(List<TestUserSpecification> testUsers) throws Exception {
     // create the profile and dataset
     super.setup(testUsers);
 
     ApiClient datasetCreatorClient = DataRepoUtils.getClientForTestUser(datasetCreator, server);
     RepositoryApi repositoryApi = new RepositoryApi(datasetCreatorClient);
 
+    if (cloudPlatform.equals(CloudPlatform.AZURE)) {
+      azureClient = AzureAuthUtils.getClient(tenantId, subscriptionId);
+      RequestRetryOptions retryOptions =
+          new RequestRetryOptions(RetryPolicyType.EXPONENTIAL, 3, 60, null, null, null);
+      blobIOTestUtility =
+          new BlobIOTestUtility(
+              AzureAuthUtils.getAppToken(tenantId), "tdrtestdatauseast", null, retryOptions);
```

Comment:
Should `tdrtestdatauseast` be a test parameter?  I forget how we do it on GCP

Reply:
This is the storage account containing the uploaded source data files, so I pulled it out into a constant! It's hard-coded for GCP as well